src/ui/view-panel.ts

```diff
@@ -78,9 +78,30 @@ class ViewPanel extends Container {
         showGridRow.append(showGridLabel);
         showGridRow.append(showGridToggle);
 
+        // show bound
+
+        const showBoundRow = new Container({
+            class: 'view-panel-row'
+        });
+
+        const showBoundLabel = new Label({
+            text: 'Show Bound',
+            class: 'view-panel-row-label'
+        });
+
+        const showBoundToggle = new BooleanInput({
+            type: 'toggle',
+            class: 'view-panel-row-toggle',
+            value: true
+        });
```

Comment:
Any reason that you're not using `LabelGroup`: https://api.playcanvas.com/classes/PCUI.LabelGroup.html

Reply:
Yep the best of reasons - I didn't know of its existence. :) I will consider using this for future updates (though in some cases it's easier to start from a css-blank-sheet rather than wrestle pcui styling).